src/components/journeys/JourneyInstancesDataTable/getStaticColumns.tsx

```diff
@@ -1,95 +1,313 @@
-import { GridColDef } from '@mui/x-data-grid-pro';
+import { IntlShape } from 'react-intl';
+import {
+  GridCellParams,
+  GridCellValue,
+  GridColDef,
+  GridFilterItem,
+} from '@mui/x-data-grid-pro';
 
+import FilterValueSelect from './FilterValueSelect';
 import JourneyInstanceTitle from 'components/journeys/JourneyInstanceTitle';
 import PersonHoverCard from 'components/PersonHoverCard';
 import ZetkinDateTime from 'components/ZetkinDateTime';
 import ZetkinPerson from 'components/ZetkinPerson';
 import ZetkinRelativeTime from 'components/ZetkinRelativeTime';
 import {
   ZetkinJourneyInstance,
+  ZetkinJourneyMilestoneStatus,
   ZetkinPerson as ZetkinPersonType,
 } from 'types/zetkin';
 
+const includes = (item: GridFilterItem) => {
```

Comment:
This function does not perform a check whether something includes something else (which I think is what the name implies) but actually **creates a function** that performs that check. 

I would call it something like `makeIncludes()` or something, but it might be that this is not even required when we reuse code elsewhere and raise the level of abstraction one step.

Reply:
aaaah "make" is the prefix to use - i thought about this and could not come up with what a good different name, after you mentioned it :)

Reply:
I tend to use `make` for these, but this is a matter of taste. This is a variation of the _Factory Function_ design pattern, sometimes referred to as "currying". I found an article that explains it well enough here: https://dev.to/qgabe/functional-programming-design-patterns-part-2-factory-method-pattern-e19 :)